spec/index.bs

```diff
@@ -139,6 +142,33 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
     </pre>
 
     In the above example <code>getMessagesFromOutbox</code> and <code>removeMessagesFromOutbox</code> are developer-defined functions.
+
+    Requesting a <a>periodic Background Sync opportunity</a> at a 60 second intervals from a <a>browsing context</a>:
```

Comment:
at a 60 second interval

Reply:
phrased as "at 60 second intervals" because the task is periodic.